### DISCUSSION TITLE
Scheduling multiple tasks on same job cluster

### DISCUSSION THREAD

**Discussion by HiteshDream** [databrickslabs/dbx#403] [Q&A] [answered]
I have a doubt and need some help. Is it possible to schedule multiple tasks on the same job cluster in dbx and how do I pass parameters separately for each task via "--parameters" option in "dbx launch" command.

This is a wonderful tool for my needs just need to figure out a couple of details before taking databricks in production.

**Comment by renardeinside** [comment, marked as answer]
hi @HiteshDream , 

This question is about two separate things - one is about multiple jobs sharing same job cluster, another is about parameter passing. 

As for **shared job cluster**, please take a look at [this documentation section ](https://www.databricks.com/blog/2022/02/04/saving-time-and-costs-with-cluster-reuse-in-databricks-jobs.html) - it's called cluster reuse feature. 

You can find an example multitask job with cluster reuse [here](https://dbx.readthedocs.io/en/latest/guidance/multitask_jobs.html#sample-multitask-jobs-based-on-jobs-api-2-1).

As for passing launch parameters at the `dbx launch` time, this functionality is currently not supported (although it will be supported in the upcoming 0.7.X release which we hopefully will deliver on 22nd of August).
